manifests/experiences/twitter/twitter.json

```diff
@@ -41,7 +41,8 @@
     },
     {
       "customPipeline": "targetingTree",
-      "visualizations": ["sunburst-targeting"],
+      "visualizations": ["Sunburst.vue"],
```

Comment:
I don't know enough about how this is done but why keep the `.vue`?

Reply:
The ".vue" extension tells us what kind of visualization to use. If we were more consistent, the others would have a ".vega" extension

Reply:
In UnitQuery we are differentiating between Vega and D3 graphs by doing `this.vizNames.filter(n => n.endsWith('.vue'))`. Could be done differently but I don't see a reason to change it now, let's wait until all vega graphs are replaced by D3